### PR TITLE
DOC: Updated Readme to reflect support for block devices

### PR DESCRIPTION
**Pull request by gotiniens**
Update `setup/Readme.md` to reflect new support for block devices

### CODE REVIEW

File: setup/Readme.md [hunk 1]
```diff
@@ -94,7 +94,10 @@ wkz run
 
 ## Background
 
-Whenever you connect your Garmin device to your Raspberry Pi, workoutizer will automatically mount the device using
-`udev`. Since it is mounted as a [gvfs](https://en.wikipedia.org/wiki/GVfs) device, the file system of your device will
+Whenever you connect your Garmin device to your Raspberry Pi, workoutizer will automatically mount the device. Workoutizer currently supports devices with MTP and Block modes. Some devices support both modes, some only one. 
+
+The MTP devices are mounted using `udev`. Since it is mounted as a [gvfs](https://en.wikipedia.org/wiki/GVfs) device, the file system of your device will
```
Comment (fgebhart) [inline setup/Readme.md:99]: `udev` is used in both cases. `udev`-rules are triggered by system events and are used to post to the workoutizer api endpoint, which in turn mounts the device. Only after the endpoint was called, your implemented logic will  distinguish between MTP and Block devices and then uses either
* `gio` for mounting MTP devices, or
* `pmount` for mounting block devices.

Could you try to rephrase this sentence a bit to reflect this?

**Review by fgebhart — CHANGES_REQUESTED** [review]
Thanks for improving the docs!

I am thinking whether it would make sense to add a list of "supported devices". Of course it would be hard to guarantee the support, because we don't have automated tests for the specific devices, but it could be helpful for people getting started.
Or maybe we could just add our two models behind "block" and "MTP" such that people are able to conclude whether their device might be rather a MTP or a block device. Something like
```md
Workoutizer currently supports devices with MTP (e.g. FR645) and Block (e.g. 920xt) modes.
```
What do you think?

**Comment by gotiniens** [comment]
> Thanks for improving the docs!
> 
> I am thinking whether it would make sense to add a list of "supported devices". Of course it would be hard to guarantee the support, because we don't have automated tests for the specific devices, but it could be helpful for people getting started.
> Or maybe we could just add our two models behind "block" and "MTP" such that people are able to conclude whether their device might be rather a MTP or a block device. Something like
> 
> ```lisp
> Workoutizer currently supports devices with MTP (e.g. FR645) and Block (e.g. 920xt) modes.
> ```
> 
> What do you think?

I think it would be better to have an list of (theoraticlly) supported devices divided into MTP and Block devices. I searched online for an list of which Garmin devices uses which connection type. Unfortunatly I cloud not find such an list. Maybe you know where we can find it?

In case we can not find such an list, we should list devices we know work with workoutizer, as there are 2 at the current moment, just naming them in the sentence is fine with me.

File: setup/Readme.md (hunk 1 above)
Comment (gotiniens) [reply setup/Readme.md]: I rephrased the sentence.

**Review by fgebhart — CHANGES_REQUESTED** [review]
> In case we can not find such an list, we should list devices we know work with workoutizer, as there are 2 at the current moment, just naming them in the sentence is fine with me.

I couldn't find such a list either. Apparently garmin prefers people to stick with plain garmin connect instead of tinkering around 😉 DC rainmaker sometimes mentions whether a device is mounted via MTP or not in his reviews, but we would have to go through maaaany reviews. Thus I would suggest to go with just mentioning the two devices somewhere in this paragraph. 

Could you add it?

**Comment by gotiniens** [comment]
DC Rainmaker has an [feature comparison](https://www.dcrainmaker.com/product-comparison-calculator?comparison=yes&ids=21805,70554,43940,122896,40809,69229,87894,114877,102074,27679,40806,57146,66436,98315,126396,3088,19938,51129,60067,117382,3086,37439,72457,97965,126393,101089,21905,21312,115951,25838,90444,120683,87895,127007,3101,34445,34446,72322,66656,35652,66324) but it does not include MTP/Block. I will add the suggested change.

**Review by fgebhart — APPROVED** [review]
perfect!

**Comment by fgebhart** [comment]
failing test is unrelated, seems to be flaky

**Comment by fgebhart** [comment]
> DC Rainmaker has an [feature comparison](https://www.dcrainmaker.com/product-comparison-calculator?comparison=yes&ids=21805,70554,43940,122896,40809,69229,87894,114877,102074,27679,40806,57146,66436,98315,126396,3088,19938,51129,60067,117382,3086,37439,72457,97965,126393,101089,21905,21312,115951,25838,90444,120683,87895,127007,3101,34445,34446,72322,66656,35652,66324) but it does not include MTP/Block. I will add the suggested change.

thanks for that link btw, wasn't aware of this great overview 😮

**Pull request merged**